WNPRC_Purchasing/test/src/org/labkey/test/tests/wnprc_purchasing/WNPRC_PurchasingTest.java

```diff
@@ -192,12 +192,80 @@ public void preTest()
     }
 
     @Test
-    public void testPurchasingLandingPage()
+    public void testCreateRequest()
+    {
+        goToProjectHome();
+        clickButton("Create Request");
+        CreateRequestPage requestPage = new CreateRequestPage(getDriver());
+
+        log("Verifying the validation for mandatory fields");
+        requestPage.submitForReview();
+        checker().verifyEquals("Invalid error message", "Unable to submit request, missing required field(s).", requestPage.getAlertMessage());
```

Comment:
This is now pluralized based on no. of errors:
So, if one error then "Unable to submit request, missing required field."
More than one then "Unable to submit request, missing required fields."

Reply:
will correct the error message.

Reply:
ive updated it, since im on this fb